--- k9copy-1.2.0/src/k9copy.desktop.orig	2007-11-01 14:47:49.000000000 +0100
+++ k9copy-1.2.0/src/k9copy.desktop	2007-11-01 14:52:20.000000000 +0100
@@ -6,9 +6,11 @@
 Comment=DVD9 to DVD5 backup tool
 Comment[fr]=Backup de DVD9 en DVD5
 Comment[tr]=DVD9 DVD5 yedekleme aracı
+Comment[pl]=Kopiowanie DVD9 do DVD5
 GenericName=Video DVD Backup
 GenericName[fr]=Backup de DVD Video
 GenericName[tr]=DVD Vidyo Yedekleme Aracı
+GenericName[pl]=Kopiowanie DVD9 do DVD5
 MimeTypes=
 Terminal=false
 Icon=k9copy.png
