--- src/k9copy.desktop.orig	2006-06-22 22:43:34.000000000 +0200
+++ src/k9copy.desktop	2006-07-04 00:00:08.185652088 +0200
@@ -6,6 +6,10 @@
 GenericName=Video DVD Backup
 Comment[fr]=Backup de DVD9 en DVD5
 GenericName[fr]=Backup de DVD Video
+Comment[pl]=Kopiowanie DVD9 do DVD5
+GenericName[pl]=Kopiowanie DVD9 do DVD5
 MimeTypes=
 Terminal=false
-Icon=k9copy.png
\ Brak znaku nowej linii na koñcu pliku
+Icon=k9copy.png
+Categories=Qt;KDE;AudioVideo;DiscBurning;
+Encoding=UTF-8
